cms/djangoapps/cms_user_tasks/signals.py

```diff
@@ -0,0 +1,50 @@
+"""
+Receivers of signals sent from django-user-tasks
+"""
+import logging
+from urlparse import urljoin
```

Comment:
This moved in Python 3; may want to import from `six.moves.urllib.parse` here to simplify porting later.